CgfConverter/Renderers/Wavefront/Wavefront.Material.cs

```diff
@@ -72,10 +72,14 @@ public void WriteMaterial(CryEngine cryEngine)
 
                         // TODO: More filehandling here
 
-                        if (!this.Args.TiffTextures)
-                            textureFile = textureFile.Replace(".tif", ".dds");
-                        else
+                        if (this.Args.PngTextures)
+                            textureFile = textureFile.Replace(".dds", ".png");
+                        else if (this.Args.TiffTextures)
                             textureFile = textureFile.Replace(".dds", ".tif");
+                        else
+                            // Is this right? exported `tif` files might actually be `tif`s, maybe
+                            // normalize by checking if the file exists first?
```

Comment:
So... I'm not really sure.  The material files all seem to reference .tif files (haven't seen any others, so if this is wrong please let me know).  The purpose of swapping out the extension is so that if you select .dds, the library_images element in the Collada file references the .dds version:

`<init_from>mobiglas_civilian_01_01_ddn.dds</init_from>`

Most Cryengine games (aka everything but SC ☹️) has the material file with a .tif extension, but the game files are all .dds.  So the purpose of this arg flag is to say "if you converted the image files to .tif and want the Collada file to use the .tif file...select tif".  If not, default to .dds.  

That said, there is a reason to convert some .dds files to .tif even for non-SC games.  For example, MWO uses different color channels for the normals file, so the normals don't work right in Blender.  Converting those image files to .tif fixes that issue.  Even opening the image in GIMP and saving again fixes it.  Super annoying.

I guess the purpose of this flag is to tell the converter which extension to use for the Collada files.  I think this will work properly as is.

Reply:
All fair points. It was mainly the arbitrary re-assignment from `.tif` -> `.dds` without ensuring the existed with either extension. I went ahead and abstracted the new texture file resolution function into `Utils/FileHandlingExtensions` so it could be used by any renderer which should handle that situation nicely, and still defaults back to `.dds` if it can't be found on disk.

As an aside, I tested `-obj` output to make sure new code worked, but none of the textures had a vaild `texture.Map` and therefore never outputed a texture line to the `obj`. Don't know if this is a new/known issue, but looks to be outside anything I've modified.